unidist/core/backends/mpi/core/controller/api.py

```diff
@@ -49,6 +51,56 @@
 topology = dict()
 # The global variable is responsible for if MPI backend has already been initialized
 is_mpi_initialized = False
+threads = []
+BACKOFF = 0.001
+exitFlag = False
+
+
+def _getopt_backoff(options):
+    backoff = options.get("backoff")
+    if backoff is None:
+        backoff = BACKOFF
+    return float(backoff)
+
+
+class Backoff:
+    def __init__(self, seconds=BACKOFF):
+        self.tval = 0.0
+        self.tmax = max(float(seconds), 0.0)
+        self.tmin = self.tmax / (1 << 10)
+
+    def reset(self):
+        self.tval = 0.0
+
+    def sleep(self):
+        time.sleep(self.tval)
+        self.tval = min(self.tmax, max(self.tmin, self.tval * 2))
+
+
+class myThread(threading.Thread):
+    def __init__(self, threadID, name, comm):
```

Comment:
```suggestion
    def __init__(self, thread_id, name, comm):
```
Use snake case for vars everywhere.